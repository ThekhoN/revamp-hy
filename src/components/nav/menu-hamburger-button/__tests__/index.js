import React from 'react';
import renderer from 'react-test-renderer';
import MenuHamburgerButton from '../index';

describe('MenuHamburgerButton', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <MenuHamburgerButton />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
