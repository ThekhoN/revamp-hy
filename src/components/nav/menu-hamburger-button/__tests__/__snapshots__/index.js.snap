// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuHamburgerButton renders correctly 1`] = `
<div
  className="float--left mobile-ui--only"
  onClick={undefined}
>
  <span
    className="svg-icon-wrapper"
  >
    <svg
      aria-labelledby="title"
      height="48"
      viewBox="0 0 48 48"
      width="48"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title
        id="title"
      >
        Homeyantra Navigation Menu
      </title>
      <path
        d="M17.1 24.9h13.4c.5 0 .9-.4.9-.9s-.4-.9-.9-.9H17.1c-.5 0-.9.4-.9.9s.4.9.9.9zM17.1 20.3h13.4c.5 0 .9-.4.9-.9s-.4-.9-.9-.9H17.1c-.5 0-.9.4-.9.9s.4.9.9.9zM30.5 27.7H17.1c-.5 0-.9.4-.9.9s.4.9.9.9h13.4c.5 0 .9-.4.9-.9s-.4-.9-.9-.9z"
        fill="#38afc7"
      />
    </svg>
  </span>
</div>
`;
